sdc/tests/test_series.py

```diff
@@ -3168,6 +3167,14 @@ def test_series_str2str(self):
                 pd.testing.assert_series_equal(hpat_func(S), test_impl(S),
                                                check_names=check_names)
 
+    def test_series_capitalize_str_with_none(self):
```

Comment:
Looks like you could combine tests `test_series_capitalize_str_with_none` and `test_series_capitalize_str` somehow. That also can be applied for other added tests.